src/config.rs

```diff
@@ -190,6 +199,8 @@ impl Default for TermvizConfig {
                 (input::ZOOM_OUT.to_string(), "-".to_string()),
                 (input::INCREMENT_STEP.to_string(), "k".to_string()),
                 (input::DECREMENT_STEP.to_string(), "j".to_string()),
+                (input::NEXT.to_string(), "n".to_string()),
```

Comment:
This is not consistent with the image mode. Maybe also use next and previous in that mode additionally?

Reply:
I thought about it but I was not sure if it was in scope of the PR. I'll do it